cerebrium/logging/logging_intro.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: "External Monitoring Tools"
+description: "Logging your predictions to external monitoring tools."
+---
+
+# External Monitoring Tools
+Cerebrium allows you to log your custom model predictions to external ML monitoring tools. Currently, we support [Censius](https://censius.ai/),
+with are planning to add support for [Arize](https://arize.com/) in Q1 2023. If there is a tool you would like to see be integrated into the platform, please contact us!
```

Comment:
"If there is a tool you would like to see be integrated into the platform" -> "If there is a tool you would like integrated into the platform"